packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -210,7 +210,6 @@ const CrudToolbar: React.FunctionComponent<CrudToolbarProps> = ({
           onApply={onApplyClicked}
           onReset={onResetClicked}
           showExplainButton={enableExplainPlan}
-          insights={insights}
```

Comment:
Please clean-up query bar code using this prop too too now that this prop is completely unused